conftest.py

```diff
@@ -13,3 +16,18 @@ def pytest_runtest_makereport(item, call):
             allure.attach(driver.get_screenshot_as_png(),
                           name=item._pyfuncitem.name,
                           attachment_type=allure.attachment_type.PNG)
+
+
+@pytest.fixture(scope='class')
+def issues(request):
+    id_issue = []
+    for i in summary:
+        response = request_api.create_issue(JsonGenerator.create_issue(i))
+        if response.status_code == 201:
```

Comment:
It would be better to add IF and collect all responses not equal to 201 and then print them in order to get detailed information for troubleshooting technical issues.